worker.py

```diff
@@ -141,6 +143,10 @@ def run(self):
                 logger.warning('Malformed response received!')
                 self.error_code = 'RESTART'
                 self.restart()
+            except BannedAccount:
+                self.error_code = 'BANNED?'
+                time.sleep(60)
```

Comment:
This is not necessary, `self.restart` already makes worker sleep. You should define higher values than usual though - (sleep_min=30, sleep_max=90 I'd say?).